### ISSUE TITLE
RFE: Make successful build message more specific

### ISSUE THREAD

**Issue by tyll** [packit/packit-service#439]
The message for successful Copr builds looks like this:

```
Congratulations! One of the builds has completed. :champagne:

You can install the built RPMs by following these steps:

* `sudo yum install -y dnf-plugins-core` on RHEL 8
* `sudo dnf install -y dnf-plugins-core` on Fedora
* `dnf copr enable packit/nmstate-nmstate-852`
* And now you can install the packages.

Please note that the RPMs should be used only in a testing environment.
```
from https://github.com/nmstate/nmstate/pull/852#issuecomment-591921005

This could be more specific:
- It mentions Fedora but Fedora is not part of the configured chroots
- It mentions only RHEL 8 but it should also work for CentOS and probably also RHEL/CentOS 7 with the correct command
- It does not mention how to install/update the packages but the information should also be available to packit.

**Comment by lachmanfrantisek** [comment]
@tyll thanks for the suggestion!

> * It mentions Fedora but Fedora is not part of the configured chroots
> 
> * It mentions only RHEL 8 but it should also work for CentOS and probably also RHEL/CentOS 7 with the correct command

That makes a lot of sense. We need to write lines that makes sense for the defined chroots.

>     * It does not mention how to install/update the packages but the information should also be available to packit.

We need to think about how we can easily get the package names.

**Comment by lachmanfrantisek** [comment]
Related to the following issues: #581, #539

**Comment by lachmanfrantisek** [comment]
We plan to move the installation info to the result page and deprecate the comment.
That task is covered by #539 but I still found this issue valid.

Regarding the package-list, we are blocked by the Copr API. We already have an issue for getting that info and using it in the Testing Farm API: #642 .

**Comment by TomasTomecek** [comment]
Is this still relevant since we no longer post that comment?

**Comment by lachmanfrantisek** [comment]
Isn't this relevant to the content of the result page?

**Comment by TomasTomecek** [comment]
oh right, that makes sense, I understand now, this is about making the text actually smart

**Comment by lachmanfrantisek** [comment]
OK, some updates:

* The content has been moved to the result page (without any change).
* The result pages are being deprecated in favour of the view on our dashboard (e.g.: https://dashboard.packit.dev/results/copr-builds/400). (Currently, the relevant part is the same.)

But some action items are still there and still valid:
* [ ] List the packages to be installed (still blocked on #642 )
* [ ] Mention only relevant OSes. (Maybe would be nice to do on the dashboard side.)

**Comment by lachmanfrantisek** [comment]
Stale bot! Why?

**Comment by lachmanfrantisek** [comment]
To provide another update, the status pages have been replaced by the dashboard ones (currently on stage, will be on prod on Monday) and Copr folks are currently working on the problem with the packages to be installed so we can start working on that part soon.

**Comment by lachmanfrantisek** [comment]
I've create a new issue for the package NVR in the install instructions: packit/dashboard#118

This is now only about using only relevant OS in the instructions.

I am moving this issue to the dashboard repository where it belongs now.
